interfaces/ServiceWorker.idl

```diff
@@ -0,0 +1,3 @@
+partial interface ServiceWorkerGlobalScope {
```

Comment:
There must be a bug here, https://w3c.github.io/ServiceWorker/ has loads of IDL.

Reply:
Looks to be the case.

Reply:
Now fixed it seems.

Reply:
This seems incorrectly include the example-only IDL from https://w3c.github.io/ServiceWorker/#extensibility? Neither this nor the previous definition are normative (both of them are inside class=example and data-no-idl blocks, which at least tells bikeshed that it shouldn't consider them to be part of the normative spec text).

Reply:
Thanks for reviewing carefully enough to notice.

@mdittmer  - Is the scraper a little over-eager here?

Reply:
That could be. Our [exclusion criteria](https://github.com/GoogleChromeLabs/webidl-diff/blob/8e6fa545cb68fada4861853d56d520d28bb357da/lib/org/chromium/webidl/IDLFragmentExtractor.js#L72) should be detecting the `example` CSS class name here. Perhaps we should file an issue.

Reply:
Address by https://github.com/GoogleChromeLabs/webidl-diff/pull/107